package.json

```diff
@@ -107,7 +107,7 @@
     "sass-loader": "^10.0.5",
     "semver": "7.3.2",
     "source-map-explorer": "^2.5.3",
-    "ssv-keys": "^0.0.18",
+    "ssv-keys": "https://github.com/bloxapp/ssv-keys.git#v4",
```

Comment:
Should not forget to change it to latest npm version when tested

Reply:
++